Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,64 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+
+A ) the code would never end. the value of n would accelerate away from
+"a" linearly i believe. the while loop could be considered n^3 and the inner loop 
+its n^2+a which if subtracted reaches O(n) now I also have to consider "a" so maybe the time complexity is O(n + a)?  
+
```

Comment:
Yes you were right there. The only thing is when we add it is constant time so we can "ignore" it. So this would be `O(n)